tests/cover/test_conjecture_engine.py

```diff
@@ -753,13 +755,19 @@ def f(data):
 
 
 def test_saves_negated_examples_in_covering():
+    """Check that every tag that's a key in examples_by_tags is either the
+    universal tag or a Negated of some other tag that is also in the dict?"""
     def f(data):
         if data.draw_bits(8) & 1:
             data.add_tag('hi')
 
     runner = ConjectureRunner(f)
     runner.run()
-    assert len(runner.target_selector.examples_by_tags) == 3
+
+    tags = set(runner.target_selector.examples_by_tags)
```

Comment:
Following @DRMacIver's [suggestion](https://github.com/HypothesisWorks/hypothesis-python/pull/1128/files/d9ea0359d7cdf47722d2bc5ae4f7ae438587a9e4#r169244529):

> Possibly the thing to do would be to update it to assert that the tags in `examples_by_tag` are either `universal` or a `Negated` of some other tag that is also in the dict?

That improvement is pretty independent of this PR (the new test passes in `master` too - I checked), except that the PR makes the old version fail because the magic number is wrong.

Reply:
It's OK to include some incidental refactoring in each pull - in this case, the fact that you had this issue makes it 'related enough' to fix IMO 😄

(I might take this too far, but #1129 has *multiple commits* of incidental fixes)